handbook/ce/onboarding.md

```diff
@@ -27,7 +27,7 @@ Welcome to Sourcegraph! This document will guide you through customer engineerin
   - Product introduction (@christina, 1 hour)
     - Optionally request access to [Figma](https://www.figma.com/) in the #product channel on Slack
   - Sourcegraph demo (@dan, 30 minutes)
-  - Product deployment (@stephen, 30 minutes)
+  - Product deployment (@gonza, 30 minutes)
```

Comment:
@pecigonzalo I swapped me with you here, for context I have only done like ~3 of these and they have been very ad-hoc, super-informal general conversations just speaking at a high level about the different deployment types and issues we face at a high level with deploying Sourcegraph at customer environments. I usually just screen share and walk through my ["Deployment types: A crash course in Sourcegraph"](https://docs.google.com/presentation/d/1u4mbXjubQqV-6WFbuS7Q1b_X6BVh-_GWzzFQMcrAzLw/edit#slide=id.g8f2bc2be97_0_107) slides with the person and answer any questions they have.

I need to link my slides somewhere so you can easily find them in the future (not sure where?) but does that sound fine to you otherwise?